README.md

```diff
@@ -2,10 +2,17 @@
 
 A module for Terasology.
 
-## Planed content
+## How to play?
 
-A character called gooey will appear. He offers you existing and dangerous quests. However be warned: He might modify your world not in a way you like it!
+First you have to start the game with GooeysQuests activated and in a Perlin world. When the game is created, you will see a character named Gooey that will ask you something. Depending on your answer, Gooey will change the world adding some structures. Now there are two available structures, the dungeon and the dwarf hall, but they have diferents combinations, so every time gooey add the structure, you will see something new. But be careful, maybe you don’t like these changes.
+
+##How to add more structures?
```

Comment:
You've missed a space between "##" and "How"

Reply:
oh, ok ,thank you